fire/inspectutils.py

```diff
@@ -100,8 +103,6 @@ def GetFullArgSpec(fn):
     # 1. If it's a builtin, it can take args.
```

Comment:
Thanks for finding the fix.

Can you update the comments to reflect your changes?

Also, are there any builtins that shouldn't return FullArgSpec(varargs='vars', varkw='kwargs')?

Reply:
I moved the comments to reflect the fact that the built-ins are handled earlier.

To be compatible with Python 3.6 behavior, all built-ins that failed the `inspect.getargspec()` should return `FullArgSpec(varargs='vars', varkw='kwargs')`. By moving the code up, my assumption is that all built-ins would fail the call, but that's just my assumption.

Reply:
I did some investigation into isbuiltin, getargspec, and getfullargspec. Here's some of what I learned:

A list of Python 2 builtin functions is available here: https://docs.python.org/2/library/functions.html
There are 76 items in the list.
inspect.isbuiltin only returns True for 48 of them, returns False for 27 of them, and cannot be called on 1 of them (print).

For Python 3 the list of builtin functions is at https://docs.python.org/3/library/functions.html
There are 68 items on this list. inspect.isbuilting gives True for 41, False for 26, and cannot be run on 1 (breakpoint).

For many of these builtins, getargspec and getfullargspec successfully return the argspec of the builtin functions. For these, we want to use those argspecs. So, it sounds to me like the behavior is actually better in 3.7 than in 3.6, and we should update the tests to allow the improved behavior rather than updating the code to enforce the worse behavior.